h2o-bindings/bin/gen_python.py

```diff
@@ -513,6 +513,22 @@ def levelone_frame_id(self):
             if "levelone_frame_id" in model and model["levelone_frame_id"] is not None:
                 return model["levelone_frame_id"]
             print("No levelone_frame_id for this model")         
+            
+        def stacking_strategy(self):
+            model = self._model_json["output"]
+            if "stacking_strategy" in model and model["stacking_strategy"] is not None:
+                return model["stacking_strategy"]
+            print("No stacking strategy for this model")  
+        
+        # Override train method to support blending 
+        def train(self, x=None, y=None, training_frame=None, blending_frame=None, **kwargs):
```

Comment:
please tell if any concern for usage of **kwargs here, as I didn't want to duplicate all params again to forward them to super impl.